### ISSUE TITLE
length of an array, and elements of arguments object outside strict mode, are not actually accessor properties, but rather proxied data properties

### ISSUE THREAD

**Issue by lewisje** [BonsaiDen/JavaScript-Garden#341]
**TL;DR:** It might be better to say they *act like* getters and setters, with a footnote explaining they're even more exotic than that.

The `length` property of an array, and the elements of the `arguments` object, are actually data properties:

```javascript
Object.getOwnPropertyDescriptor([], 'length');
//=> Object {value: 0, writable: true, enumerable: false, configurable: false}
(function () {return Object.getOwnPropertyDescriptor(arguments, 0);})(undefined); // passing in no arguments returns just `undefined`
//=> Object {value: undefined, writable: true, enumerable: true, configurable: true}
```

The behavior of both of these things can be approximated by an object wrapped in an ES6 `Proxy` but maybe talking about that would be too much, considering most of this guide is not written for ES6; also, to my understanding, the link outside strict mode between the `arguments` object and named arguments cannot be so approximated, because the scope object that the named arguments are properties of cannot be accessed via script.
